indico/modules/events/surveys/models/surveys.py

```diff
@@ -135,7 +168,7 @@ def event(self, event):
     def has_ended(self):
         return bool(self.end_dt) and self.end_dt <= now_utc()
 
-    @property
+    @hybrid_property
```

Comment:
I've added a new commit to upstream wip/evaluation that fixes those hybrid properties properly. When rebasing again you'll probably have to resolve a conflict and/or ensure that nothing weird happens here.